mm2src/coins/nft/storage/wasm/wasm_storage.rs

```diff
@@ -194,7 +585,9 @@ impl TableSignature for NftListTable {
                 &["chain", "token_address", "token_id"],
                 true,
             )?;
+            table.create_multi_index(Self::CHAIN_BLOCK_NUMBER_INDEX, &["chain", "block_number"], false)?;
```

Comment:
Is it possible for a chain to have different blocks with the same `block_number` if no, `table.create_multi_index(Self::CHAIN_BLOCK_NUMBER_INDEX, &["chain", "block_number"], false)?;` should be true

Reply:
Well, it is an index for NFTs. It is possible to receive multiple nfts in the same block in one chain.